x-pack/plugins/infra/public/alerting/log_threshold/log_threshold_rule_type.ts

```diff
@@ -11,11 +11,11 @@ import { ObservabilityRuleTypeModel } from '../../../../observability/public';
 import {
   LOG_DOCUMENT_COUNT_ALERT_TYPE_ID,
   PartialAlertParams,
```

Comment:
@ersin-erdal Shall we rename this to `PartialRuleParams`?

Reply:
As i said in the Pr description i didnt touch the alerting types with this PR to not to make it too big.
There is already another PR to tackle that: https://github.com/elastic/kibana/issues/120486